tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1214,6 +1492,12 @@ everflow/test_everflow_per_interface.py::test_everflow_per_interface[ipv6-erspan
       - "platform in ['x86_64-8800_lc_48h_o-r0', 'x86_64-8800_lc_48h-r0']"
       - "(is_multi_asic==True) and https://github.com/sonic-net/sonic-buildimage/issues/11776"
 
+everflow/test_everflow_per_interface.py::test_everflow_per_interface[ipv6-erspan_ipv6-default:
+  skip:
+    reason: "Skip for IPv6-only topologies"
```

Comment:
this one looks to be ipv6, maybe skipped by mistake?

Reply:
Change to xfail. Thanks